### DISCUSSION TITLE
Deployment by invalid-email-address 1 min ago on website

### DISCUSSION THREAD

**Discussion by ArmeniaViolin** [vercel/community#119] [Help] [answered]
![Screenshot from 2021-12-30 14-14-05](https://user-images.githubusercontent.com/96684678/147742738-ea06c00f-9f20-471e-b4d1-27ca4bf76336.png)
I am wondering, why do this happen?

**Comment by Pieparker** [comment, marked as answer]
Hey! Perhaps the deploy was triggered by someone with that username? If that doesn't explain matters, could you provide the deployment ID or commit SHA so we can take a look?
